argilla-server/src/argilla_server/api/handlers/v1/records.py

```diff
@@ -171,10 +191,16 @@ async def upsert_suggestion(
 
     # NOTE: If there is already a suggestion for this record and question, we update it instead of creating a new one.
     # So we set the correct status code here.
+    action = "create"
     if await Suggestion.get_by(db, record_id=record_id, question_id=suggestion_create.question_id):
         response.status_code = status.HTTP_200_OK
+        action = "update"
```

Comment:
Maybe we can have an `upsert` action so you don't need to add logic trying to know if it's a `create` or an `update`?

Reply:
We can also rename it to "upsert" but I wanted to avoid capturing all edge cases "publish", "search", "read", "list" "upsert" "update" because I thought it might be a bit much for metrics/telemetry. @frascuchon @jfcalvo if you feel it would help development, I can make a finer distinguishment.